src/pybind/mgr/telemetry/module.py

```diff
@@ -742,6 +742,7 @@ def handle_command(self, inbuf, command):
             r = {}
             for opt in self.MODULE_OPTIONS:
                 r[opt['name']] = getattr(self, opt['name'])
+                r['last_upload'] = time.ctime(self.last_upload) if self.last_upload else self.last_upload
```

Comment:
@yaarith could you move this assignment statement out of the `for` loop?

Reply:
oops, sorry. thanks, @tchaikov!